src/vueFileImporter.ts

```diff
@@ -189,22 +192,39 @@ async function insertComponent(
     if (syntax!.id === 'js') {
       match2 = /export[\s]*default[\s]*\{/.exec(text)
     } else if (syntax!.id === 'ts') {
-      match2 = /export[\s]*default.+\.extend[\s]*\(\{/.exec(text)
+      match2 = /(export[\s]*default.+\.extend[\s]*\(\{)|(export[\s]*default[\s]*defineComponent\(\{)/.exec(
+        text
+      )
```

Comment:
Updated this to match the `defineComponent()` function call

Reply:
thanks for your vue3 support.